src/Bridge/Symfony/Bundle/Resources/config/swagger.xml

```diff
@@ -6,34 +6,141 @@
 
     <services>
 
-        <service id="api_platform.swagger.normalizer.documentation" class="ApiPlatform\Core\Swagger\Serializer\DocumentationNormalizer" public="false">
-            <argument type="service" id="api_platform.metadata.resource.metadata_factory" />
-            <argument type="service" id="api_platform.metadata.property.name_collection_factory" />
-            <argument type="service" id="api_platform.metadata.property.metadata_factory" />
-            <argument type="service" id="api_platform.resource_class_resolver" />
-            <argument type="service" id="api_platform.operation_method_resolver" />
-            <argument type="service" id="api_platform.operation_path_resolver" />
-            <argument type="service" id="api_platform.router" />
-            <argument type="service" id="api_platform.filters" />
-            <argument type="service" id="api_platform.name_converter" on-invalid="ignore" />
+        <!-- Extractors -->
+        <service id="api_platform.swagger.extractor.swagger_extractor"
+                 alias="api_platform.core.swagger.extractor.route_documentation_extractor"/>
 
-            <tag name="serializer.normalizer" priority="16" />
+        <service class="ApiPlatform\Core\Swagger\Extractor\RouteExtractor"
```

Comment:
I think that you can make this service private (`public="false"`) to improve performance.

The `id` should come first  and use a self closing XML tag.

Generally speaking (for all XML files), we prefer to put the whole XML tag on only one line.